core/src/main/java/de/danoeh/antennapod/core/feed/Feed.java

```diff
@@ -419,6 +420,24 @@ public void setImageUrl(String imageUrl) {
         return items;
     }
 
+    public List<FeedItem> dedupItems() {
```

Comment:
I would move this to FeedHandler because it is only used there and not useful in other places. No need to make this available to all users of the class.

Reply:
done